README.md

```diff
@@ -403,7 +403,7 @@ For example:
 ```yml
 ci:
   allow_failures:
-    - ci/circleci
+    - buildkite/:pipeline_name
```

Comment:
maybe `buildkite/<pipeline_name>` is more clear?

Reply:
I prefer going with a colon (especially in routes) as it bandwagons the [rails routing param syntax](http://guides.rubyonrails.org/routing.html).
There are two places where the `<placeholder_var>` is used in the `README`.
If you think it is better to keep consistency with the rest of the document I will change this.